csaf_2.1/prose/edit/src/schema-elements-01-defs-03-full-product-name.md

```diff
@@ -239,9 +239,10 @@ Two `*` MUST NOT follow each other.
     IC25T060ATCS05-0
 ```
 
-##### Full Product Name Type - Product Identification Helper - purl
+##### Full Product Name Type - Product Identification Helper - purls
 
-The package URL (purl) representation (`purl`) is a `string` of 7 or more characters with `pattern` (regular expression):
+List of purls (`purls`) of value type `array` with 1 or more items contains a list of package URL (purl) identifiers.
+A purl is a `string` of 7 or more characters that meets the criteria of `pattern` (regular expression):
```

Comment:
Please make sure that aligns with other sections that use an array, e.g. remediations or notes.

Reply:
Not sure I understand. In what way should the text align with those other sections?

Reply:
I think it does already (the text translation of the schema constraints into English prose) ... I compared the two text lines to the notes section introductory prose.